modules/users/server/controllers/users/users.profile.server.controller.js

```diff
@@ -54,15 +55,16 @@ exports.update = function (req, res) {
 exports.changeProfilePicture = function (req, res) {
   var user = req.user;
   var message = null;
+  var upload = multer({ dest: './modules/users/client/img/profile/uploads/' }).single('newProfilePicture');
```

Comment:
I did this way because I was having an issue when I needed to configure another upload path for another route.
Basically, when was an middleware in ``` config/lib/express.js ```, all uploads goes to the ```/uploads``` folder. If I needed to create another resource in my API where I upload a file and want to store in a different folder, I wasn't able to do it using multer.
Also, I agree with you that we can put the destination path in env, but, if some people, like me, want to add new destinations, how we can configure to support many destinations? Any suggestion?